application/src/main/java/com/sanction/lightning/resources/FacebookResource.java

```diff
@@ -72,7 +67,21 @@ public Response getUser(@Auth Key key, @QueryParam("username") String username)
               .entity("'username' query parameter is required for getUser").build();
     }
 
-    PilotUser pilotUser = thunderClient.getUser(username);
+    PilotUser pilotUser;
+    try {
+      pilotUser = thunderClient.getUser(username);
+    } catch (RetrofitError e) {
+      // Returns the response thunder gave
```

Comment:
I would change this comment to say `If the Error has a null response, then Thunder is down.`

And then add a comment above the `return` saying `Otherwise, return the response that Thunder gave.`